crates/cli/src/lib.rs

```diff
@@ -86,7 +89,9 @@ impl Args {
                     }
 
                     // start signing server
-                    tokio::spawn(SigningService::run(config.chain, signer_config));
+                    // TODO: generate jwt for each module id
+                    let jwts = HashMap::new();
```

Comment:
If I understand correctly: `jwts` should contain all the module JWTs **and** the `pbs_jwt`, right?